rakelib/plugin.rake

```diff
@@ -61,6 +61,8 @@ namespace "plugin" do
     Dir["logstash-core*.gem"].each do |gem|
       rm(gem)
     end
```

Comment:
it will only glob .gem files in the current path.... what is scary? (maybe I'm missing something?)